test/integration/user_install/controls/user_install_spec.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
```

Comment:
May I asked why are tests being duplicated here?
I see you've made changes to `/test/integration/user_isntall/controls/user_install.rb`, is that not enough?

Reply:
I tried to merge Lance's rebase into the version I was working.  Mucked things up good. Sorry, I'll fix things.